src/nunit.xamarin/nunit.xamarin.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Company>NUnit</Company>
+    <Product>NUnit 3 Xamarin Runner</Product>
+    <Copyright>Copyright (C) 2018, Rob Prouse</Copyright>
+    <Title>NUnit.Xamarin</Title>
+    <Description>NUnit 3 Test Runner for Xamarin</Description>
+    <Version Condition="'$(Version)' == ''">3.10.1</Version>
+
+    <!-- NuGet Package Info -->
+    <PackageId>nunit.xamarin</PackageId>
+    <Title>NUnit 3 Xamarin Runner</Title>
+    <Authors>Rob Prouse</Authors>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/nunit/nunit.xamarin/master/LICENSE.txt</PackageLicenseUrl>
+    <PackageIconUrl>https://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_256.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/nunit/nunit.xamarin</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/nunit/nunit.xamarin</RepositoryUrl>
+    <PackageTags>nunit xamarin android ios monoandroid monotouch tdd unit test testing</PackageTags>
+    <Description>
+Adds support for running NUnit 3 unit tests on Xamarin
+
+Supported Xamarin platforms:
+- Android
+- iOS
+- Windows 10 Universal Apps
+    </Description>
```

Comment:
There's still duplicate Title and Description elements in here.

Reply:
Fixed, should be ready to merge once CI passes and you approve @ChrisMaddock